pkg/security/podsecuritypolicy/provider_test.go

```diff
@@ -113,11 +113,12 @@ func TestCreatePodSecurityContextNonmutating(t *testing.T) {
 
 func TestCreateContainerSecurityContextNonmutating(t *testing.T) {
 	// Create a pod with a security context that needs filling in
+	untrue := false
 	createPod := func() *api.Pod {
 		return &api.Pod{
 			Spec: api.PodSpec{
 				Containers: []api.Container{{
-					SecurityContext: &api.SecurityContext{},
+					SecurityContext: &api.SecurityContext{RunAsNonRoot: &untrue},
```

Comment:
the point of this test is to make sure CreateContainerSecurityContext doesn't mutate a pod, so I'd want to leave it completely empty (no RunAsNonRoot set) to enable as many of the `if ... == nil { ... }` blocks as possible to run